docs/migration-guide.md

```diff
@@ -0,0 +1,159 @@
+
+- `<%REPEAT:5, Repeat Me!%>`
+
+# `GOTOBLOCK` and `GRAPH` are now parameters.
+
+Related to the sub commands bullet above, the GOTOBLOCK and GRAPH sub commands have been removed in favor of being used as blocks.
+`GOTOBLOCK` should simply be specified as the second argument for its two supported commands, like this:
+
+- `<%OPENPAGE:08-27-2021,GOTOBLOCK 2%>`
+- `<%SIDEBARWINDOWOPEN:08-27-2021,GOTOBLOCK 2%>`
+
+Similarly, `GRAPH` should be specified as the second argument to its one supported command:
+
+- `<%SIDEBARWINDOWOPEN:08-27-2021,GRAPH%>`
+
+Additionally, `GOTOBLOCK` now supports any numerical value, not just 1 or -1.
+
+# `42Setting` Removed
+
+A goal of creating the SmartBlocks extension was so that it could live independently, absent the original Roam42 extension. This means removing any Roam42 specific functionality. This resulted in the removal of the `42Setting` command.
+This command can be replaced using the existing SmartBlocks command set. For example, to replace a block that had this:
+
+- `<%42SETTING:name%>`
+- Can be replaced with this block:
+
+- `<%REPLACE:<%REPLACE:<%RESOLVEBLOCKREF:<%BLOCKMENTIONS:1,42Setting,{uid},name%>%>,<%HASHTAG:42Setting%>%>, name %>`
+
+It will be supported when the SmartBlocks extension is in included with the Roam42 bundle.
+
+# `roam42` methods
+
+If you used `JAVASCRIPT` commands that contain `roam42.common` methods, these methods will no longer be available by default with SmartBlocks v2. Here are a list of common Roam42 methods with their mitigation nested under it:
+
+- `roam42.common.createBlock(parent_uid, order, block_string)`
+
+```javascript
+const uid = window.roamAlphaAPI.util.generateUID();
+window.roamAlphaAPI.createBlock({
+  location: { "parent-uid": parent_uid, order },
+  block: { string: block_string, uid },
+});
+return uid;
+```
+
+- `roam42.common.createSiblingBlock(fromUID, block_string, bBelow)`
+
+```javascript
+const [{ order }, { uid: parent_uid }] = window.roamAlphaAPI.q(`[:find 
+(pull ?b [:block/order]) (pull ?p [:block/uid])
+:where 
+[?b :block/uid "${fromUID}"]
+[?p :block/children ?b]
+]`)[0];
+const uid = window.roamAlphaAPI.util.generateUID();
+window.roamAlphaAPI.createBlock({
+  location: { "parent-uid": parent_uid, order: order + bBelow },
+  block: { string: block_string, uid },
+});
+return uid;
+```
+
+- `roam42.common.moveCursorToNextBlock`
+  - This method is pretty hacky and would not recommend using until Roam releases an API for doing this. In any case, if you're still interested, I would look at [the source code directly](https://github.com/dvargas92495/roam42/blob/master/common/commonFunctions.js#L317-L330).
+- `roam42.common.moveCursorToPreviousBlock`
+  - This method is pretty hacky and would not recommend using until Roam releases an API for doing this. In any case, if you're still interested, I would look at [the source code directly](https://github.com/dvargas92495/roam42/blob/master/common/commonFunctions.js#L332-L344).
```

Comment:
Where are these files located?

Reply:
You'll have to go back into the history for workbench, and use a commit snapshot (which is what I should've done)